data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
 		"bunks" 245
 		"mass" 940
```

Comment:
Oh yes, jack the price up by at least 50%,  they aren't going to be cheap.  Also if possible set it up so that you actually get the ship about 3 months after you order with a significant deposit on order.  Full payment is of course due on delivery.  But if you wait too long after it's been constructed you loose your deposit and the ship gets sold to someone else.